### PR TITLE
feat(frontend/settings): theme tool

### LINKED ISSUE #506: Integrate theme-tool
In overlay settings add a button that links to theme-tool.
Update: Tutorials should also be linked ShabadOS/tutorials#9

### PR DESCRIPTION
**Pull request by saihaj**
### Summary of PR

Adds a button in overlay settings to link to theme tool and also changes the description.

**Before**
image

**After**
image

### Time spent on PR
30 minutes

### Linked issues
Fix #506 

### Reviewers
@Harjot1Singh

### CODE REVIEW

File: app/frontend/src/Settings/OverlaySettings.js [hunk 1]
```diff
@@ -37,16 +37,24 @@ const OverlaySettings = () => {
               When a line is activated in the controller and seen on the presenter,
               the same information is used to generate the overlay.
               Choose what to display and how to display it with the options below.
-              Currently only one overlay configuration is available.
+              You can choose from a variety of pre-configured overlays or
+              create your own using theme tool.
```
Comment (Harjot1Singh) [inline app/frontend/src/Settings/OverlaySettings.js:41]:
```suggestion
              create your own using the theme tool.
```

But try to make sure the text doesn't overflow to another line if possible.

**Comment by bhajneet** [comment]
There is a very clear 2-column pattern of key & values. To have two buttons designed like this is confusing to the pattern. Please use one button per row/line.

**Comment by bhajneet** [comment]
Additionally, the theme tool could just be in the Learn More button, no? Keep it simple and use the existing paradigms in place for advanced learning/usage.

**Comment by saihaj** [comment]
Should we revert this back and make the change?

**Comment by Harjot1Singh** [comment]
I’d suggest PR’ing the change (since dev isn’t a release, reverting doesn’t serve a point if we’re going to recommit).

The Learn More button provides a lot of information that a user may not ever know about, and that the theme tool has no link to. So, it would make sense to still keep both buttons.

**Pull request merged**